AntistasiOfficial.Altis/REINF/addToGarrison.sqf

```diff
@@ -0,0 +1,156 @@
+private ["_posicionTel","_cercano","_cosa","_grupo","_unidades","_salir"];
+openMap true;
+posicionTel = [];
+_cosa = _this select 0;
+
+onMapSingleClick "posicionTel = _pos";
+
+hint "Select the zone on which sending the selected troops as garrison";
+
+waitUntil {sleep 0.5; (count posicionTel > 0) or (not visiblemap)};
+onMapSingleClick "";
+
+if (!visibleMap) exitWith {};
+
+_posicionTel = posicionTel;
+
+_cercano = [markers,_posicionTel] call BIS_fnc_nearestPosition;
+
+if !(_posicionTel inArea _cercano) exitWith {hint "You must click near a marked zone"};
+
+if (not(_cercano in mrkSDK)) exitWith {hint "That zone does not belong to Syndikat"};
+
+if ((_cercano in puestosFIA) and !(isOnRoad getMarkerPos _cercano)) exitWith {hint "You cannot manage garrisons on this kind of zone"};
+
+_cosa = _this select 0;
+
+_grupo = grpNull;
+_unidades = objNull;
+
+if ((_cosa select 0) isEqualType grpNull) then
+	{
+	_grupo = _cosa select 0;
+	_unidades = units _grupo;
+	}
+else
+	{
+	_unidades = _cosa;
+	};
+
+_salir = false;
+
+{
+if ((typeOf _x == guer_POW) or (typeOf _x in CIV_units) or (!alive _x)) exitWith {_salir = true}
+} forEach _unidades;
+
+if (_salir) exitWith {hint "Static crewman, prisoners, refugees or dead units cannot be added to any garrison"};
+
+if ((groupID _grupo == "MineSw") or (groupID _grupo == "Watch") or (isPlayer(leader _grupo))) exitWith {hint "You cannot garrison player led, Watchpost, Roadblocks or Minefield building squads"};
+
+
+if (isNull _grupo) then
+	{
+	_grupo = createGroup side_blue;
+	_unidades joinSilent _grupo;
+	hint "Adding units to garrison";
+	{arrayids pushBackUnique (name _x)} forEach _unidades;
+	}
+else
+	{
+	hint format ["Adding %1 squad to garrison", groupID _grupo];
+	};
+
+_garrison = [];
+_garrison = _garrison + (garrison getVariable [_cercano,[]]);
+{_garrison pushBack (typeOf _x)} forEach _unidades;
+garrison setVariable [_cercano,_garrison,true];
+[_cercano] call mrkUpdate;
+
+_noBorrar = false;
+
+if (spawner getVariable _cercano != 2) then
```

Comment:
Its because of this. Spawn system in Altis is based on bool, WotP has three possible states.